examples/lb3-application/lb3app/test/acceptance.js

```diff
@@ -0,0 +1,130 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-lb3-application
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+const lb3App = require('../server/server');
+const request = require('@loopback/testlab').supertest;
+const assert = require('assert');
+const ExpressServer = require('../../dist/server').ExpressServer;
+require('should');
+
+let app;
+
+function json(verb, url) {
```

Comment:
nitpick: the function name is a bit confusing, maybe call it as `setContentType`?
And why the test has to set the content type and Accept as `application/json` then send the payload? Is it also part of the migration guide?
(hmm, "migration" may not be the accurate word haha)

Reply:
I thought this was traditional lb3 naming because it's what I've seen in lb3 examples (like [here](https://github.com/strongloop/loopback-example-app-logic/blob/master/test/rest_api_test.js#L12-L17) and [here](https://github.com/strongloop/loopback-example-access-control/blob/master/test/rest_api_test.js#L12-L17)) 🙈 

> And why the test has to set the content type and Accept as application/json then send the payload? Is it also part of the migration guide?

Since we want users to change as little code as possible, I maintained the function as is, but I agree we can rename the function name 😄

Reply:
Renamed to `request` to match what the function does; what do you think?

Reply:
I see, I didn't realize it's from LB3...then I am ok to keep the name as it is. Since it's migration.